manager/manager.go

```diff
@@ -769,11 +769,35 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// As we are proposing having multiple ways to detect a container (in addition to cgroups)
+// need the ability to delete
```

Comment:
Ok, update the comment with that explanation? :)
I think the comment is not a proper sentence.